plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -527,6 +527,12 @@ public function utilityController_exportProfiles_create($sender) {
             ->where('u.Deleted', 0)
             ->where('u.Admin <', 2);
 
+        if (val('DateOfBirth', $fields)) {
```

Comment:
As an FYI: val() is a bit of overkill in this case. Using `!empty($fields['DateOfBirth'])` would be better.